deepmd/kernels/cute/neo/readout_l0.py

```
@@ -0,0 +1,774 @@
+
+def _readout_input_fold_cache_matches(
+    output_ffn: Any,
+    sources: tuple[torch.Tensor, ...],
+    cache_key: tuple[tuple[Any, ...], ...],
+) -> bool:
+    cache = getattr(output_ffn, _READOUT_INPUT_FOLD_CACHE, None)
+    return (
+        cache is not None
+        and len(cache) == 2
+        and len(cache[0]) == len(sources)
+        and all(cached is current for cached, current in zip(cache[0], sources))
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)